modules/persistent-storage-csi-snapshots-delete.adoc

```diff
@@ -24,6 +21,47 @@ metadata:
 driver: hostpath.csi.k8s.io
 deletionPolicy: Delete <1>
 ----
-<1> If the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.
+<1> When deleting the volume snapshot, if the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.
```

Comment:
This is existing content and might be out of scope of this PR, but we've been avoiding future tense (e.g., will be). I'll leave it up to you if you want to change it. :)

Reply:
Good catch! I :) updated two instances using future tense in this note.